velox/common/encode/Base64.h

```diff
@@ -57,10 +57,9 @@ class Base64 {
 
   static std::string decode(folly::StringPiece encoded);
 
-  /// Returns decoded size for the specified input. Adjusts the 'size' to
-  /// subtract the length of the padding, if exists.
-  static size_t
-  calculateDecodedSize(const char* data, size_t& size, bool withPadding = true);
+  /// Returns the actual size of the decoded data. Will also remove the padding
```

Comment:
nit: enclose the variable name in single quotes like
` /// length from the input data 'size'.`

Reply:
Updated the code